Upgrade to Bootstrap 5.1

simon04 opened the pull request:
https://getbootstrap.com/docs/5.1/migration/

![Screenshot 2022-05-20 at 23-16-48 Wikidata Image Positions](https://user-images.githubusercontent.com/782446/169611896-ab985c08-966a-46ab-9a02-229c6e1a90cf.png)

simon04 committed:
Upgrade to Bootstrap 5.1

https://getbootstrap.com/docs/5.1/migration/

lucaswerkmeister commented on templates/base.html:
```diff
@@ -5,20 +5,22 @@
     <title>{% block title %}Wikidata Image Positions{% endblock title %}</title>
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <link rel="stylesheet" href="https://tools-static.wmflabs.org/cdnjs/ajax/libs/twitter-bootstrap/4.6.0/css/bootstrap.min.css" integrity="sha384-B0vP5xmATw1+K9KRQjQERJvTumQW0nPEzvF6L/Z6nronJ3oUOFUFpCjEUQouq2+l" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://tools-static.wmflabs.org/cdnjs/ajax/libs/twitter-bootstrap/5.1.3/css/bootstrap.min.css" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
```
Maybe we should change the name from `twitter-bootstrap` to just `bootstrap` while we’re at it? (Both seem to serve the exact same file, so I’m guessing `twitter-bootstrap` is the older name.)

simon04 replied:
Good idea. Done.

lucaswerkmeister reviewed (commented):
These buttons don’t look so great anymore:
![image](https://user-images.githubusercontent.com/2346599/169690142-25e60170-aefc-4ad7-a7e0-5bd41ba0f27e.png)
Although to be fair, I only changed those to use `ml-2` (now `ms-2` apparently?) on Friday (eeb434ac0b) ^^

I’m not sure how I feel about the centered navbar-nav, but it’s probably okay.
![image](https://user-images.githubusercontent.com/2346599/169690195-c0088fd0-2472-4466-a5fb-46371027f2c3.png)

simon04 committed:
cdnjs: bootstrap

simon04 committed:
ms-2

lucaswerkmeister commented:
> I’m not sure how I feel about the centered navbar-nav, but it’s probably okay. ![image](https://user-images.githubusercontent.com/2346599/169690195-c0088fd0-2472-4466-a5fb-46371027f2c3.png)

Ah, but your screenshot makes me realize that the documentation/toolforge/source part moves around depending on whether login is enabled or not (it’s either centered or on the end side, depending on whether there is an authentication area or not), that’s probably not ideal… I guess if OAuth isn’t configured, the authentication area should be something like an empty div instead of nothing at all? (I’m assuming that it looks the same as long as OAuth is configured, regardless of whether the user is logged in or not, but haven’t tried that out yet.)

simon04 committed:
me-auto me-1

simon04 commented:
Sorry, I missed the `mr-*` which are now called `me-*`. Should be fixed.

![image](https://user-images.githubusercontent.com/782446/169714164-08a9693f-633f-4742-9674-000f8c275171.png)

lucaswerkmeister committed:
Add class="form-label" to <label>s

This is apparently required in Bootstrap 5, and in practice seems to add
some vertical margin between the label and input.

lucaswerkmeister commented:
Deployed and looking good, thank you!
